src/scan_modules/mod.rs

```diff
@@ -10,6 +10,7 @@ pub mod unicode_fingerprinting;
 pub fn make_default_scan_modules() -> Vec<Box<ScanModule>> {
     vec![
         Box::new(strings::StringsScanModule {}),
+        Box::new(unicode_fingerprinting::FingerprintScanModule {}),
```

Comment:
Would you consider renaming this unit struct to `UnicodeFingerprintScanModule` to match the filename? Thanks!